includes/admin/class-wc-rest-payments-survey-controller.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Class WC_REST_Payments_Survey_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * REST controller for settings.
+ */
+class WC_REST_Payments_Survey_Controller extends WP_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v3';
+
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payments/survey';
+
+	/**
+	 * Configure REST API routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			[
+				'methods'             => WP_REST_Server::CREATABLE,
+				'callback'            => [ $this, 'submit_survey' ],
+				'permission_callback' => [ $this, 'check_permission' ],
+				'args'                => [],
+			]
+		);
+	}
+
+	/**
+	 * @param WP_REST_Request $request the request being made.
+	 *
+	 * @return WP_REST_Response
+	 */
+	public function submit_survey( WP_REST_Request $request ) {
+		$wpcom_request = Client::wpcom_json_api_request_as_user(
+			'/marketing/survey',
+			'v2',
+			array(
+				'method'  => 'POST',
+				'headers' => array(
+					'Content-Type'    => 'application/json',
+					'X-Forwarded-For' => $this->get_current_user_ip(),
+				),
+			),
+			$request->get_json_params()
```

Comment:
Just grab the responses and add them to the request. site id, survey id, etc can be passed from the backend